Resources/Locale/en-US/pneumatic-cannon/pneumatic-cannon-component.ftl

```diff
@@ -0,0 +1,29 @@
+﻿### Loc for the pneumatic cannon.
+
+pneumatic-cannon-component-verb-gas-tank-name = Eject gas tank
+pneumatic-cannon-component-verb-eject-items-name = Eject all items
+
+## Shown when inserting items into it
+
+pneumatic-cannon-component-insert-item-success = You insert { THE($item) } into { THE($cannon) }.
+pneumatic-cannon-component-insert-item-failure = You can't seem to fit { THE($item) } in { THE($cannon) }.
+
+## Shown when changing the fire mode or power.
+
+pneumatic-cannon-component-change-fire-mode = You set the firing mode to: { $mode }.
+pneumatic-cannon-component-change-power = You set the power to: { $power }.
```

Comment:
I have some suggestions for more detailed texts here.

```suggestion
pneumatic-cannon-component-change-fire-mode = { $mode ->
    [All] You loosen the valves to fire everything at once.
    *[Single] You tighten the valves to fire one item at a time.
}
pneumatic-cannon-component-change-power = { $power ->
    [High] You set the limiter to maximum power. Beware the recoil.
    [Medium] You set the limiter to medium power.
    *[Low] You set the limiter to low power.
}

```

Reply:
yeah i wanted to make these more detailed but couldnt think of anything creative, thanks